Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -3912,6 +4136,256 @@ def screenshot_url_command(client: Client, args: dict[str, Any]) -> CommandResul
+
+    Returns:
+        CommandResults: JSON response of added indicator tags.
+    """
+    feed_uuid = args.get("feed_uuid")
+    if not feed_uuid:
+        raise ValueError("Feed UUID is required")
+
+    indicator_name = args.get("indicator_name")
+    if not indicator_name:
+        raise ValueError("Indicator name is required")
+
+    if not args.get("tags"):
+        raise ValueError("The 'tags' argument is required and cannot be empty.")
+
+    result = client.add_indicators_tags(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.AddIndicatorTags",
+        outputs_key_field="feed_uuid",
+        outputs=result,
+        readable_output=tableToMarkdown("SilentPush Add Indicator Tags", result),
+        raw_response=result,
+    )
+
+
+@metadata_collector.command(
+    command_name="silentpush-run-threat-check",
+    inputs_list=RUN_THREAT_CHECK_INPUTS,
+    outputs_prefix="SilentPush.RunThreatCheck",
+    outputs_list=RUN_THREAT_CHECK_OUTPUTS,
+    description="This command runs the threat check on the specified ",
+)
+def run_threat_check_command(client: Client, args: dict[str, Any]) -> CommandResults | dict:
+    """
+    Command handler to fetch threat checks.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'feed_uuid' key.
+
+    Returns:
+        CommandResults: JSON response of threat check.
+    """
+    threat_type = args.get("type")
+    if not threat_type:
+        raise ValueError("Threat type is required")
+
+    data = args.get("data")
+    if not data:
+        raise ValueError("Data is required")
+
+    user_identifier = args.get("user_identifier")
+    if not user_identifier:
+        raise ValueError("User identifier is required")
+
+    query = args.get("query")
+    if not query:
+        raise ValueError("Query is required")
+
+    result = client.run_threat_check(args)
+
+    return CommandResults(
+        outputs_prefix="SilentPush.RunThreatCheck",
+        outputs_key_field="query",
+        outputs=result,
+        readable_output=tableToMarkdown("SilentPush Threat Check", result),
+        raw_response=result,
+    )
+
+
+@metadata_collector.command(
+    command_name="silentpush-get-data-exports",
+    inputs_list=GET_DATA_EXPORTS_INPUTS,
+    outputs_prefix="SilentPush.GetDataExports",
+    outputs_list=[],
+    file_output=True,
+    description="This command runs the threat check on the specified ",
+)
+def get_data_exports_command(client: Client, args: dict[str, Any]) -> dict[str, Any]:
+    """
+    Command handler to export data.
+
+    Args:
+        client (Client): SilentPush API client instance.
+        args (Dict[str, Any]): Command arguments, must include 'feed_uuid' key.
+
+    Returns:
+        CommandResults: JSON response of threat check.
+    """
+
+    feed_url = args.get("feed_url")
+
+    if not feed_url:
+        raise ValueError("Feed URL is required")
```

Comment:
if the arg is set to required in the yml, no need for this check, the command will not run without it.